subgraph/tests/mockTxReceipt.ts

```diff
@@ -109,23 +109,66 @@ export function newTransactionReceipt(
     defaultBigInt,
     defaultBigInt,
     defaultAddress,
-    [
-      createCancelKeyEventLog(
-        BigInt.fromU32(tokenId),
-        Address.fromString(lockOwner),
-        Address.fromString(keyOwnerAddress),
-        refund > BigInt.fromU32(0) ? defaultIntBytes : defaultZeroIntBytes
-      ),
-      // This Log shouldn't be there if the tokenAddress is nullAddress but id does not really matter
-      createTransferEventLog(
-        tokenAddress,
-        Address.fromString(lockAddress),
-        Address.fromString(keyOwnerAddress),
-        refund > BigInt.fromU32(0) ? defaultIntBytes : defaultZeroIntBytes
-      ),
-    ],
+    logs,
     defaultBigInt,
     defaultAddressBytes,
     defaultAddressBytes
   )
 }
+
+// Create transaction receipt for mock transaction
+export function newCancelKeyTransactionReceipt(
+  tokenAddress: Address,
+  refund: BigInt
+): ethereum.TransactionReceipt {
+  return newTransactionReceipt([
```

Comment:
Ohhh! That makes sense. I am not sure why I assumed there could be a single "log" per mocked transaction.